arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -956,6 +962,16 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		096A55DC1FD8677800D2E91B /* Raster layer (URL) */ = {
+			isa = PBXGroup;
+			children = (
+				09B7FDD21FD86FD200B4DFC1 /* RasterLayerUsingURLVC.swift */,
+				096A55DF1FD8677800D2E91B /* RasterLayerUsingURL.storyboard */,
+			);
+			name = "Raster layer (URL)";
```

Comment:
Name should be Raster layer (service)

Reply:
There are other samples where we follow `Some layer (URL)` format for online services. Would this still be `(service)`?

Reply:
Yeah, I think service is the way to go. You're right, we have used URL in the past, in many of those cases you do pass a URL to the constructor, so it does kind of fit. But not this one. If we had to re-think all previous ones, I think a strong case could be made for using service in all